### PR TITLE
refactor(swap): update swap shape

### PR DESCRIPTION
**Pull request by acdibble**

### CODE REVIEW

File: packages/sdk/src/swap/v2/types.ts [hunk 1]
```diff
@@ -47,27 +47,39 @@ interface DepositFields {
...
+      executedAt: number;
+      executedBlockIndex: string;
+      retryCount: number;
+    } | null;
+    currentChunk: {
+      inputAmount: string;
+      scheduledAt: number;
+      scheduledBlockIndex: string;
+      retryCount: number;
+    } | null;
```
Comment (niklasnatter) [inline packages/sdk/src/swap/v2/types.ts:79]: not sure if this can be null

File: packages/swap/src/routes/v2/swap.ts [hunk 2]
```diff
@@ -149,32 +148,36 @@ router.get(
...
-      if (a.swapExecutedAt && b.swapExecutedAt) {
-        return a.swapExecutedAt.valueOf() - b.swapExecutedAt.valueOf() ? 1 : -1;
-      }
-      return a.swapExecutedAt && !b.swapExecutedAt ? 1 : 0;
-    });
+    let originalInputAmount;
+
+    if (swapRequest) {
+      originalInputAmount = swapRequest.fees.reduce(
+        (acc, fee) => (fee.asset === swapRequest.srcAsset ? acc.minus(fee.amount) : acc),
```
Comment (niklasnatter) [inline packages/swap/src/routes/v2/swap.ts:155]: might need to filter to only subtract ingress fee

File: packages/swap/src/routes/v2/swap.ts [hunk 3]
```diff
@@ -183,6 +186,7 @@ router.get(
     );
 
     const aggregateFees = rolledSwaps?.fees
+      .splice(0)
```
Comment (niklasnatter) [inline packages/swap/src/routes/v2/swap.ts:189]: 🤔

**Review by niklasnatter — APPROVED** [review]
looks nice 🙌

**Review by zoheb391 — APPROVED** [review]
think we're ready for beta?

File: packages/swap/src/routes/v2/swap.ts (hunk 2 above)
Comment (acdibble) [reply packages/swap/src/routes/v2/swap.ts:155]: for older swaps the broker fee too

File: packages/sdk/src/swap/v2/types.ts (hunk 1 above)
Comment (acdibble) [reply packages/sdk/src/swap/v2/types.ts:79]: it can be null if the entire DCA swap is completed

**Pull request merged**